crates/crypto/addresses/src/address/resource_address.rs

```diff
@@ -38,6 +38,20 @@ impl ResourceAddress {
     }
 }
 
+impl TryFrom<(ScryptoManifestResourceAddress, NetworkID)> for ResourceAddress {
+    type Error = CommonError;
+    fn try_from(
+        value: (ScryptoManifestResourceAddress, NetworkID),
```

Comment:
an alternative spelling is: `(address, network): (ScryptoManifestResourceAddress, NetworkID)` which I've started to use